### DISCUSSION TITLE
Two way communication

### DISCUSSION THREAD

**Discussion by prerak9** [jgromes/RadioLib#324] [Q&A]
Hello,
I am using the following hardware - 
1. Arduino UNO - 2
2. Lora Sx1278 Module - 2

Voltage - 3.3V

I want to establish a two way communication between the two devices where the receiver sends a feed back to transmitter that data is successfully received.

I want both devices to be able to communicate to each other.
I couldn't find any such example code, so please help me with a sample code or any relevant info.
Thank You

**Comment by jgromes** [comment]
You seem to be using a device with 3.3V logic (SX1278) with a 5V Arduino (Uno). That will destroy your SX1278 in the longterm and lead to stability issues in the short term. You have to use a logic level shifter between them.

> I couldn't find any such example code

But I assume you did find the examples for transmit and receive right? Soooo ... have you tried combining them in some way? There's also a handy example for SX126x that shows a ping-pong exchange, it should be very easy to modify for SX1278.

EDIT: Also, this is still not an issue - converting to discussion instead.

**Comment by jgromes** [comment]
Regarding your comment in https://github.com/jgromes/RadioLib/issues/325:

You can just call the `transmit()` method after you succesfully `receive()` a packet. Have you tried that?

**Comment by prerak9** [comment]
Yes, Did not work

**Reply by jgromes** [reply]
"Did not work" - in what way?

And can you post the code?
